<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="ES-SYS-656" name="com.easyproctor.qa.testcases.VideoPageTest" tests="4" failures="0" timestamp="22 Oct 2021 09:18:37 GMT" time="0.005" errors="1">
  <testcase name="approvetest" time="0.001" classname="com.easyproctor.qa.testcases.VideoPageTest">
    <skipped/>
  </testcase> <!-- approvetest -->
  <testcase name="setUp" time="0.000" classname="com.easyproctor.qa.testcases.VideoPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.easyproctor.qa.util.TestUtil.testngmethod(TestUtil.java:51)
	at com.easyproctor.qa.base.BasePage.initialization(BasePage.java:67)
	at com.easyproctor.qa.testcases.VideoPageTest.setUp(VideoPageTest.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="videoplaytest" time="0.003" classname="com.easyproctor.qa.testcases.VideoPageTest">
    <skipped/>
  </testcase> <!-- videoplaytest -->
  <testcase name="rejecttest" time="0.001" classname="com.easyproctor.qa.testcases.VideoPageTest">
    <skipped/>
  </testcase> <!-- rejecttest -->
</testsuite> <!-- com.easyproctor.qa.testcases.VideoPageTest -->
